src/Functions/countries.php

```diff
@@ -9,254 +9,513 @@
+            'JEY' => __('Jersey', 'mailoptin'),
+            'JOR' => __('Jordan', 'mailoptin'),
+            'KAZ' => __('Kazakhstan', 'mailoptin'),
+            'KEN' => __('Kenya', 'mailoptin'),
+            'KIR' => __('Kiribati', 'mailoptin'),
+            'KWT' => __('Kuwait', 'mailoptin'),
+            'KGZ' => __('Kyrgyzstan', 'mailoptin'),
+            'LAO' => __('Laos', 'mailoptin'),
+            'LVA' => __('Latvia', 'mailoptin'),
+            'LBN' => __('Lebanon', 'mailoptin'),
+            'LSO' => __('Lesotho', 'mailoptin'),
+            'LBR' => __('Liberia', 'mailoptin'),
+            'LBY' => __('Libya', 'mailoptin'),
+            'LIE' => __('Liechtenstein', 'mailoptin'),
+            'LTU' => __('Lithuania', 'mailoptin'),
+            'LUX' => __('Luxembourg', 'mailoptin'),
+            'MAC' => __('Macao', 'mailoptin'),
+            'MKD' => __('North Macedonia', 'mailoptin'),
+            'MDG' => __('Madagascar', 'mailoptin'),
+            'MWI' => __('Malawi', 'mailoptin'),
+            'MYS' => __('Malaysia', 'mailoptin'),
+            'MDV' => __('Maldives', 'mailoptin'),
+            'MLI' => __('Mali', 'mailoptin'),
+            'MLT' => __('Malta', 'mailoptin'),
+            'MHL' => __('Marshall Islands', 'mailoptin'),
+            'MTQ' => __('Martinique', 'mailoptin'),
+            'MRT' => __('Mauritania', 'mailoptin'),
+            'MUS' => __('Mauritius', 'mailoptin'),
+            'MYT' => __('Mayotte', 'mailoptin'),
+            'MEX' => __('Mexico', 'mailoptin'),
+            'FSM' => __('Micronesia', 'mailoptin'),
+            'MDA' => __('Moldova', 'mailoptin'),
+            'MCO' => __('Monaco', 'mailoptin'),
+            'MNG' => __('Mongolia', 'mailoptin'),
+            'MNE' => __('Montenegro', 'mailoptin'),
+            'MSR' => __('Montserrat', 'mailoptin'),
+            'MAR' => __('Morocco', 'mailoptin'),
+            'MOZ' => __('Mozambique', 'mailoptin'),
+            'MMR' => __('Myanmar', 'mailoptin'),
+            'NAM' => __('Namibia', 'mailoptin'),
+            'NRU' => __('Nauru', 'mailoptin'),
+            'NPL' => __('Nepal', 'mailoptin'),
+            'NLD' => __('Netherlands', 'mailoptin'),
+            'NCL' => __('New Caledonia', 'mailoptin'),
+            'NZL' => __('New Zealand', 'mailoptin'),
+            'NIC' => __('Nicaragua', 'mailoptin'),
+            'NER' => __('Niger', 'mailoptin'),
+            'NGA' => __('Nigeria', 'mailoptin'),
+            'NIU' => __('Niue', 'mailoptin'),
+            'NFK' => __('Norfolk Island', 'mailoptin'),
+            'MNP' => __('Northern Mariana Islands', 'mailoptin'),
+            'PRK' => __('North Korea', 'mailoptin'),
+            'NOR' => __('Norway', 'mailoptin'),
+            'OMN' => __('Oman', 'mailoptin'),
+            'PAK' => __('Pakistan', 'mailoptin'),
+            'PSE' => __('Palestinian Territory', 'mailoptin'),
+            'PAN' => __('Panama', 'mailoptin'),
+            'PAN' => __('Papua New Guinea', 'mailoptin'),
```

Comment:
Duplicate array key `PAN`.

Please always go over your work to ensure accuracy.